data/ExperimentDesignPresets.ts

```diff
@@ -16,22 +16,28 @@ interface Preset<T> {
   preset: RecursivePartial<T>;
 }
 
-const presets: { [id: string]: Preset<ExperimentDesign> } = {
+const presets: { [id: string]: Preset<ExperimentRecipe> } = {
   empty_aa: {
     name: "A/A Experiment",
     description:
       "A design for diagnostic testing of targeting or enrollment. Fixed to 1% of the population.",
     preset: {
-      proposedDuration: 28,
-      proposedEnrollment: 7,
-      branches: [
-        { slug: "control", ratio: 1, value: null },
-        { slug: "treatment", ratio: 1, value: null },
-      ],
-      bucketConfig: {
-        randomizationUnit: "normandy_id",
-        count: 100,
-        total: 10000,
+      filter_expression:
+        "env.version|versionCompare('{minFirefoxVersion}') >= 0 && {audienceFilterExpression}",
+      targeting:
+        "[{randomizationUnit}, {bucketNamespace}]|bucketSample({bucketStart}, {bucketCount}, {bucketTotal}) && {audienceTargeting}",
```

Comment:
You need quotations around `bucketNamespace`, i.e. `[{randomizationUnit}, "{bucketNamespace}"]`